release/xgboost_tests/workloads/train_small.py

```diff
@@ -16,7 +16,7 @@
 from ray.util.xgboost.release_test_util import train_ray
 
 if __name__ == "__main__":
-    ray.init(address="auto")
+    ray.client().job_name(os.environ["RAY_JOB_NAME"]).connect()
```

Comment:
`os.environ.get("RAY_JOB_NAME", "no_name")`?

Reply:
Only set `job_name()` in client mode now, and default to `train_small`.